client/resource_group/controller/controller.go

```diff
@@ -958,8 +954,12 @@ func (gc *groupCostController) collectRequestAndConsumption(selectTyp selectType
 }
 
 func (gc *groupCostController) calcRequest(counter *tokenCounter) float64 {
-	value := counter.avgRUPerSec*gc.run.targetPeriod.Seconds() + bufferRUs
-	value -= counter.limiter.AvailableTokens(gc.run.now)
+	value := counter.avgRUPerSec * gc.run.targetPeriod.Seconds() * needTokensAmplification
+	if gc.run.initialRequestCompleted {
+		value -= counter.limiter.AvailableTokens(gc.run.now)
+	} else {
+		value += counter.initialToken - counter.limiter.AvailableTokens(gc.run.now)
+	}
```

Comment:
Perhaps `value -= counter.limiter.AvailableTokens(gc.run.now)' can be extracted out.